mvvmfx-validation/README.md

```diff
@@ -0,0 +1,11 @@
+# MvvmFX Utils
+
+This module contains some validators that can be useful for JavaFX and mvvmFX developers.
```

Comment:
I would use the text "This module contains validation utilities that can be...".
This way the description is more general.